widgetssdk/src/main/java/com/glia/widgets/view/Dialogs.kt

```diff
@@ -65,6 +67,19 @@ object Dialogs {
             }
     }
 
+    private fun showDialogBasedOnView(
+        context: Context,
+        theme: UiTheme,
+        view: VisitorCodeView,
+        cancelable: Boolean
+    ): AlertDialog {
+        return MaterialAlertDialogBuilder(context)
+            .setView(view)
+            .setCancelable(cancelable)
+            .show()
+            .also { setDialogBackground(it, theme.gliaChatBackgroundColor) }
```

Comment:
What if chat background color is changed? Do we need to change this dialog color also?